packages/devtools-plugin/src/devtools.plugin.ts

```diff
@@ -47,6 +47,11 @@ export class NgxsReduxDevtoolsPlugin implements NgxsPlugin {
     }
 
     return next(state, action).pipe(
+      catchError(error => {
+        const type = getActionTypeFromInstance(action);
+        this.devtoolsExtension!.send({ ...action, type }, this.store.snapshot());
+        throw error;
+      }),
```

Comment:
Ok, this is better but what if the action that errors here was the `@@INIT` action?
The dev tool would still need to init. Should we maybe move the init code somewhere else?

Reply:
I refactored so that the same logic is shared when sending to the dev tools.